testsuite/interaction_potentials/dihedral_harmonic/test_dihedralharmonic.py

```diff
@@ -0,0 +1,130 @@
+        system.skin = skin
+        system.comm = MPI.COMM_WORLD
+        nodeGrid = espressopp.tools.decomp.nodeGrid(espressopp.MPI.COMM_WORLD.size)
+        cellGrid = espressopp.tools.decomp.cellGrid(box, nodeGrid, cutoff, skin)
+        system.storage = espressopp.storage.DomainDecomposition(system, nodeGrid, cellGrid)
+        self.system = system
+ 
+    def test_phi0(self):
+        #create system with three torsions, and for each one set a potential with a different phi0 values, to check that the interaction potential works for all combinations of positive and negative phi and phi0
+       
+        phi0 = [10.0,-170.0,170.0]
+
+        particle_list = [
+            #add particles with initial torsion angle +90
+            (1, 0, espressopp.Real3D(2.0, 3.0, 3.0), 1.0),
+            (2, 0, espressopp.Real3D(2.0, 2.0, 3.0), 1.0),
+            (3, 0, espressopp.Real3D(2.0, 2.0, 2.0), 1.0),
+            (4, 0, espressopp.Real3D(3.0, 2.0, 2.0), 1.0),
+            #add particles with initial torsion angle 160
+            (5, 0, espressopp.Real3D(2.0, 3.0, 3.0), 1.0),
+            (6, 0, espressopp.Real3D(2.0, 2.0, 3.0), 1.0),
+            (7, 0, espressopp.Real3D(2.0, 2.0, 2.0), 1.0),
+            (8, 0, espressopp.Real3D(2.4, 0.8, 2.0), 1.0),
+            #add particles with initial torsion angle -161
+            ( 9, 0, espressopp.Real3D(2.0, 3.0, 3.0), 1.0),
+            (10, 0, espressopp.Real3D(2.0, 2.0, 3.0), 1.0),
+            (11, 0, espressopp.Real3D(2.0, 2.0, 2.0), 1.0),
+            (12, 0, espressopp.Real3D(1.6, 0.8, 2.0), 1.0),
+        ]
+        self.system.storage.addParticles(particle_list, 'id', 'type', 'pos', 'mass')
+        self.system.storage.decompose()
+
+        quadrupleslist = [[1,2,3,4],[5,6,7,8],[9,10,11,12]]
+        torsiontuples = [(1,2,3,4),(5,6,7,8),(9,10,11,12)]
+        bondtuples = [(1,2),(2,3),(3,4),(5,6),(6,7),(7,8),(9,10),(10,11),(11,12)]
+
+        #add torsions
+        interactions = []
+        for i in xrange(3):
+          fql = espressopp.FixedQuadrupleList(self.system.storage)
+          fql.addQuadruples([torsiontuples[i]])
+          interaction = espressopp.interaction.FixedQuadrupleListDihedralHarmonic(self.system,fql,potential=espressopp.interaction.DihedralHarmonic(K=1.0,phi0=phi0[i]*math.pi/180.0))
+          self.system.addInteraction(interaction)
+          interactions.append(interaction)
+
+        #add bonds so that atoms in the torsions don't drift too far apart
+        fpl = espressopp.FixedPairList(self.system.storage)
+        fpl.addBonds(bondtuples)
+        interaction = espressopp.interaction.FixedPairListHarmonic(self.system,fpl,potential=espressopp.interaction.Harmonic(K=1.0,r0=1.0))
+        self.system.addInteraction(interaction)
+
+        integrator = espressopp.integrator.VelocityVerlet(self.system)
+
+        integrator.run(50)
+
+        self.assertAlmostEqual(interactions[0].computeEnergy(),0.747885,places=5)
```

Comment:
I suppose that the idea was to use it here as the check for computeEnergy calculated independently in the test?